lib/WUI/progress_data_wrapper.h

```diff
@@ -16,6 +16,8 @@ uint8_t progress_get_percentage();
 
 void progress_format_time2end(char *dest, uint16_t feedrate);
 
+uint32_t progress_time2end(uint16_t feedrate);
```

Comment:
It would be worth to document parameter and return value units.

Reply:
This whole progress_data_wrapper will be removed after we integrate changes from https://github.com/prusa3d/Prusa-Firmware-Buddy/pull/436 to WUI